pkgs/development/python-modules/trackpy/default.nix

```diff
@@ -52,6 +57,14 @@ buildPythonPackage rec {
     "trackpy"
   ];
 
+  disabledTests = [
+    # AttributeError, IndexError
```

Comment:
I'm not familiar with this package, but according to the error logs, I guess this package is not compatible with pandas 2.0 at all.